src/sentry/utils/suspect_resolutions/metric_correlation.py

```diff
@@ -55,7 +55,7 @@ def is_issue_error_rate_correlated(
 
 def calculate_pearson_correlation_coefficient(x: List[int], y: List[int]) -> int:
```

Comment:
I just realized, what're the range of expected values of the pearson R value? Should this be an `int` return type.

Reply:
It should be a `float`